test/new_relic/agent/error_filter_test.rb

```diff
@@ -88,6 +88,13 @@ def test_skip_invalid_status_codes
           assert @error_filter.ignore?(TestExceptionA.new, 401)
           assert @error_filter.ignore?(TestExceptionA.new, 500)
         end
+
```

Comment:
When writing tests, I lean toward writing one assertion per example. I understand this file isn't currently following these practices, but would you be open to refactoring the added test into its own method? This helps use the test suite as documentation (each definition provides a bit more context about what the file is trying to accomplish) and helps with debugging.

That could look something like: 
```ruby
def test_ignore_integer_status_codes
  # ... code written for your test
end 
```

Reply:
Sounds good, I also prefer one assertion per example. Thanks for suggesting a test name!